packages/user-interface/src/components/Notification.tsx

```diff
@@ -0,0 +1,44 @@
+import Alert, { AlertProps } from "@gemeente-denhaag/alert";
+import styles from "./Notification.module.scss";
+import { useEffect, useState } from "react";
+import classNames from "classnames";
+
+interface Props extends AlertProps {
+  closable?: boolean;
+}
+
+const Notification = ({
+  closable = true,
+  title,
+  text,
+  variant,
+  className,
+  ...props
+}: Props) => {
+  const [hideNotification, setHideNotification] = useState(false);
```

Comment:
Misschien handig om dit nog in de positieve zin te schrijven: showNotification?

Reply:
Goed punt!

Reply:
O ja :)